cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -240,16 +240,20 @@ fn lower_insn_to_regs<'a>(ctx: Ctx<'a>, iri: IRInst) {
         | Opcode::SshrImm => {
             panic!("ALU+imm and ALU+carry ops should not appear here!");
         }
-        Opcode::Fadd | Opcode::Fsub => {
+        Opcode::Fadd | Opcode::Fsub | Opcode::Fmul | Opcode::Fmax | Opcode::Fmin | Opcode::Fdiv => {
             let regD = output_to_reg(ctx, iri, 0);
             let regL = input_to_reg(ctx, iri, 0);
             let regR = input_to_reg(ctx, iri, 1);
             let is64 = flt_ty_is_64(ty.unwrap());
             if !is64 {
-                let inst = if op == Opcode::Fadd {
-                    SSE_Op::SSE_Addss
-                } else {
-                    SSE_Op::SSE_Subss
+                let inst = match op {
+                    Opcode::Fadd => SSE_Op::SSE_Addss,
+                    Opcode::Fsub => SSE_Op::SSE_Subss,
+                    Opcode::Fmul => SSE_Op::SSE_Mulss,
+                    Opcode::Fmax => SSE_Op::SSE_Maxss,
```

Comment:
A quick sanity check: for the min/max operations, does this provide the correct semantics for the unordered case relative to what CLIR requires?

Reply:
In order to help getting this landed, I checked, and the answer is no: in the old backend, fmax goes through `expand_minmax` which does more complicated things using the x86's min and max (also for 0 and -0, etc.). So this needs a bit of re-work.

Reply:
Been offline the last few days .. Thanks for the update. I looked into expand_minmax and at first glance I wasn't what needed to be done here so I've simply removed the min/max and will push support in another patch.